Update AD SaaS Box tutorial with correct Identifier and Reply URL

dddugan opened the pull request:
The Box tutorial is factually incorrect. Identifier value is actually for Reply URL, which was not listed at all. Add Reply URL, correct Identifier value, update matching screenshot.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dddugan committed:
Correct Box tutorial idenifier and reply url

Instructions are incorrect as written. The value shown for Identifier is actually the Reply URL, which was not included at all. Add Reply URL, correct Identifer, update screenshot to reflect these changes.

dddugan committed:
update screenshot to include Reply URL

include Reply URL and correct value for Identifier
